nbqa/__main__.py

```diff
@@ -451,6 +455,73 @@ def _run_command(
     return out, err, output_code, mutated
 
 
+def _find_config_file(
+    command: str,
+) -> Tuple[Optional[str], Optional[ConfigParser], Optional[str]]:
+    """
+    Find config file.
+
+    Parameters
+    ----------
+    command
+        Third-party tool being run.
+
+    Returns
+    -------
+    config_file
+        Name of config file.
+    config
+        ConfigParser object which has read in config file.
+    config_prefix
+        Prefix of sections in config file.
+    """
+    config_files = CONFIG_FILES
+    for config_file in config_files:
```

Comment:
We could use CONFIG_FILES directly in the for loop instead of `config_files = CONFIG_FILES`